org.lflang/META-INF/MANIFEST.MF

```diff
@@ -35,7 +35,8 @@ Export-Package: org.lflang,
  org.lflang.serializer,
  org.lflang.services,
  org.lflang.util,
- org.lflang.validation
+ org.lflang.validation,
+ org.lflang.formatting2
```

Comment:
Why are we introducing a `formatting2` package?

Reply:
If I remember correctly, I think Xtext expects to find formatting-related code in a package called `formatting2`, in the same way that it expects the validator to be in a certain package. I am not sure if this is a hard requirement though.

Reply:
OK, weird. But if Xtext is the reason, then sure :shrug: